Add Lecture 19

superslalomracer opened the pull request:
Add the lecture notes for day 19 as the Lecture_19.md file. Follow
current naming, line spacing, and formatting convention. Major topics
include bug localization and the delta debugging algorithm.

superslalomracer committed:
Add Lecture 11 Notes

Add the lecture notes for day 11 as the Lecture_11.md file. Follow
current naming, line spacing, and formatting convention. Major topics
include Test Suite Quality Metrics and various types of Code
Coverage.

superslalomracer committed:
Add Lecture 19

Add the notes for lecture 19 as the Lecture_19.md file. Follow current
naming convention, line spacing, and formatting as of template v1.
Major topics include Bug localization and Delta Debugging algorithm.

kevinaangstadt reviewed (changes requested):
This PR appears to also contain a commit for Lecture 11.  Please refactor your commits so that this PR only contains Lecture 19-related commits.  You may need to use the `-f` flag when you push to override the previous commits in this branch.

superslalomracer committed:
Delete Lecture_11.md

superslalomracer commented:
The Lecture_11.md file has been deleted from this branch. It should be fine to merge now.

superslalomracer committed:
Update and Modify Lecture_19.md

Delete the 'Overview' section from the notes and specify the language
of code for each code block in the Lecture_19.md file.

'Overview' no longer conforms to the current formatting because it is
not needed, and was addressed in issue #33. Specifying the code
language is in response to changes suggested for template.md.
